src/main/drivers/pwm_output.c

```diff
@@ -246,33 +227,39 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
 {
     memset(motors, 0, sizeof(motors));
     
-    uint32_t timerMhzCounter = 0;
     bool useUnsyncedPwm = motorConfig->useUnsyncedPwm;
 
+    unsigned usMin = 0;
```

Comment:
I think it is easier to read if it is microseconds, i.e. 125 (everyone knows it as OS125 etc) and then it is simply mhz.